cmd/fluxctl/install_cmd.go

```diff
@@ -97,6 +153,14 @@ func (opts *installOpts) RunE(cmd *cobra.Command, args []string) error {
 	}
 
 	for fileName, content := range manifests {
+		// The way the templating works, it will run through each
+		// file; but some may elide the entire content if it's not
+		// needed. So, don't bother writing anything out if the
+		// template evaluated to just whitespace.
+		if len(bytes.TrimSpace(content)) == 0 {
```

Comment:
Is this necessary? It is already done in `FillInTemplates()` (except for the trimming)

https://github.com/fluxcd/flux/blob/324d5b2d162e0b76fcaab0f6140d5ca2951ae093/pkg/install/install.go#L59-L61

(it was included in the midst of this PR)

Reply:
It is now, yes -- I created trouble for myself by fixing it while I was rebasing :-/